### ISSUE TITLE
Phantasm needs small rewording

### ISSUE THREAD

**Issue by jcte02** [openlegend/core-rules#229]
`Phantasm` reads
>all squares must touch but can be in any shape/pattern

How can we reword this to remove the notion about a grid?

**Comment by brianfeister** [comment]
I think it would talk about a number of `5' cubes` so then 

> all 5' cubes must touch but can be in any shape/pattern

**Comment by twystedcures** [comment]
What if groups are using hex grids? Could you go with 5' space?

**Comment by brianfeister** [comment]
Lol. Maybe

**Comment by istabosz** [comment]
I fixed the wording in the banes file and SRD. I basically removed that line because it added unnecessary confusion and didn't allow for things like a single Phantasm creating several distinct lights.

While I was at it, I also added the ability to move the illusion, which I think we took for granted as assumed.
